js/mod/ban-list.js

```diff
@@ -73,7 +73,12 @@ var banlist_init = function(token, my_boards, inMod) {
 	  un = "<em>"+_("system")+"</em>";
 	}
 	return pre + un + suf;
-      } }
+      } },
+      id: {
+         name: (inMod)?"Edit":"&nbsp;", width: (inMod)?"35px":"0px", fmt: function(f) {
+	 if (!inMod) return '';
+         return "<a href=?/edit_ban/"+f.id+">Edit</a>";
```

Comment:
this is XSS waiting to happen. if the complex mod_pages query breaks down i'd target this right here were i an attacker as you didn't quote the href param unlike all others in the file.